doc/btp_spec.txt

```diff
@@ -550,6 +550,18 @@ Events:
 		This event indicates that the remote Identity Address has been
 		resolved.
 
+	Opcode 0x88 - Connection Parameters event
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Connection Interval (2 octets)
+					Connection Latency (2 octets)
+					Supervision Timeout (2 octets)
+
+		This event can be sent when a connection is established or
```

Comment:
The same situation. I would send this event on parameters change only. Sending another event after GAP Connected event is a bit odd. If you really need to know connection parameters of newly established connection, just extend the GAP Connected event with those.